tests/test_pvc_cuncurrent_deletion_creation.py

```diff
@@ -0,0 +1,216 @@
+"""
+Test to verify concurrent creation and deletion of multiple PVCs
+"""
+import os
+import subprocess
+import pytest
+import yaml
+import logging
+from munch import munchify
+import ocs.defaults as defaults
+
+from ocs import ocp
+from ocs import exceptions
+from ocsci import tier1, ManageTest
+from utility import templating
+
+log = logging.getLogger(__name__)
+TEMPLATE_DIR = "templates/ocs-deployment"
+TEMP_YAML_FILE_SC = '/tmp/temp_file_sc.yaml'
+TEMP_YAML_FILE_PVC = '/tmp/temp_file_pvc.yaml'
+SC = ocp.OCP(kind='StorageClass', namespace=defaults.ROOK_CLUSTER_NAMESPACE)
+PVC = ocp.OCP(kind='PersistentVolumeClaim',
+              namespace=defaults.ROOK_CLUSTER_NAMESPACE)
+PV = ocp.OCP(kind='PersistentVolume',
+             namespace=defaults.ROOK_CLUSTER_NAMESPACE)
+
+
+@pytest.fixture(scope='class')
+def test_fixture(request):
+    """
+    Removing the resources created for this test
+    """
+    self = request.node.cls
+
+    def finalizer():
+        teardown(self)
+    request.addfinalizer(finalizer)
+
+
+def teardown(self):
```

Comment:
you don't need self here cause you are not using any class attribute

Reply:
Done